packages/components/card/stories/EntryCard.stories.tsx

```diff
@@ -27,38 +22,6 @@ export default {
   title: 'Components/Card/EntryCard',
 } as Meta;
 
-const DragHandle = (props: { drag: React.ReactElement }) => {
```

Comment:
Should we provide stories with DnD-kit instead?

Reply:
In my opinion this doesn't serve much value in Storybook, and then we have to keep these examples up do date in multiple places.

Reply:
I think it's gonna be valuable to have at least one example of DnD with our card, but not necessarily an EntryCard

Reply:
There's already drag and drop in the Card example with DragHandle: https://forma-36-git-feat-dnd-docs.colorfuldemo.com/components/card#with-a-custom-drag-handle